edn/tests/tests.rs

```diff
@@ -53,6 +53,7 @@ fn test_nil() {
 
 #[test]
 fn test_nan() {
+    assert_eq!(nan("#fNaN").unwrap(), Float(OrderedFloat(f64::NAN)));
```

Comment:
I don't believe this is correct.

`#fNaN` is itself a tag, so `#fNaN` should not match the `nan` production. Whitespace is required: `#f NaN`. This test should instead assert that `#fNaN` *does not* match `nan`, right?

@victorporof, am I misreading this change?